bigquery/tests/unit/test_table.py

```diff
@@ -1115,11 +1115,20 @@ def _get_target_class():
     def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
+    def _setUpConstants(self):
+        import datetime
+        from google.cloud._helpers import UTC
+
+        self.WHEN_TS = 1437767599.006
```

Comment:
`.006` is not accurately representable in base 2, so you may encounter rounding issues with this value. Choose a fraction such as `.125` instead.

Reply:
Made change to .125 as recommended.